handbook/engineering/search/iteration_log.md

```diff
@@ -31,6 +29,28 @@ This document contains the goals and work log for the search team's [2-week iter
     - RFC published.
 - **Work log:**
     - YYYY-MM-DD: $UPDATE
+    
+### Structural search for monorepos
+- **Owner(s):** Rijnard, Camden
+- **Outcomes:**
+    - Make structural search work better for monorepos. Currently monorepos can time out for structural search purely because it takes too long to copy the data to search. The outcome focuses on architectural changes to avoid unnecessary data zipping and copying to our searcher worker.
```

Comment:
@rvantonder @camdencheek I added this placeholder outcome as a copy-paste of a previous entry. Can you review this together & edit it to be accurate? Thanks!

Reply:
Looks great. The tracking issue for this is https://github.com/sourcegraph/sourcegraph/issues/14619, but unsure if we need to put it in the iteration log.

Reply:
```suggestion
    - Make structural search work better for monorepos. Currently monorepos can time out for structural search purely because it takes too long to copy the data to search. The outcome focuses on architectural changes to avoid unnecessary data zipping and copying to our searcher worker (see [sourcegraph/sourcegraph#14619](https://github.com/sourcegraph/sourcegraph/issues/14619)).
```
 Thanks! Adding GH issue as it is useful for discoverability for anyone who wishes to dig deeper.